README.md

```diff
@@ -10,9 +10,9 @@ making working with your database easier, but this comes at the cost of ineffici
 confusing code mapping. The goal of ScalaRelational is to provide a one-to-one representation of the SQL language in
 Scala and provide performance improvements to working with the database.
 
-Currently H2 and MySQL is tested, but other databases technically are supported though certain functionality may require
-some slight alterations to work properly. One of the goals of ScalaRelational is to be a straight-forward, type-safe framework
-to write SQL queries in Scala with no magic or guesswork of what the resulting SQL will be.
+Currently H2 and MariaDB (version 10.0) is tested, but other databases technically are supported though certain functionality
```

Comment:
Sure, as MariaDB is a "drop-in" replacement for MySQL. But the current travis test use the `mariadb` plugin (https://github.com/lummax/scalarelational/commit/64c53dbabc8ba3eb195be90470ff639674439135) so the automated tests don't check for MySQL compliance.